.circleci/config.yml

```diff
@@ -919,13 +925,19 @@ jobs:
       - checkout
       - attach_workspace:
           at: .
-      - run: scripts/bytecodecompare/storebytecode.sh && cp -v report.txt bytecode-report-osx.txt
+      - run: mkdir test-cases/
```

Comment:
This block seems to be the same  as the one above, could it be shared somehow?

Reply:
Yeah, I can parameterize it.

Reply:
@cameel do you still want to do this /\? Just wondering if I'm missing something

Reply:
I originally wanted to do this and actually tried it yesterday but in the end I'll need some trial regarding syntax and error and this is very annoying to debug (basically needs almost a complete CI run every time) so it was getting in the way of me trying to solve the problem with failing `t_bytecode_compare`. So I wanted to get back here and say that I think it's better to leave it as is in this PR and do it separately.

But also I'm no really sure if it's worth it. Only 2 out of the 4 bytecode jobs are similar and even then they're not identical. Parameterizing it won't save us that much code. I think it might be better to just leave it as is.

Reply:
Ok, fine with leaving as is.